docs/frontends/webapi.rst

```diff
@@ -1850,6 +1850,27 @@ This is the "Welcome Page", and contains a few distinct sections::
  Grid Status: introducer information, helper information, connected storage
               servers.
 
+``GET /?t=json``   (the json welcome page)
+
+This is the "json Welcome Page", and contains connectivity status
+of the introducer(s) and storage server(s), here's an example::
+
+  {
+   "introducers": {
+    "statuses": []
+   },
+   "servers": {
+    "other_nodeid": {
+     "available_space": 123456,
+     "nickname": "other_nickname \u263b",
+     "version": "",
+     "connection_status": "summary",
+     "last_received_data": ""
```

Comment:
This should be a number (seconds since epoch), not a string. And we need to decide what it returns if the server hasn't been contacted yet.. probably `null`?

It might be a good idea to add a list explaining each key, at least giving the expected type of each one.